apps/site/assets/ts/schedule/components/Map.tsx

```diff
@@ -105,9 +107,9 @@ const shouldVehicleDisplay = (
   marker: Marker,
   shapeIdsInDirection: string[]
 ): boolean => {
-  if (marker.shape_id) {
-    return shapeIdsInDirection.some(shapeId => shapeId === marker.shape_id);
-  }
+  // if (marker.shape_id) {
```

Comment:
@hannahpurcell do you still need this function if it always returns `true`?

Reply:
Lol, I commented it out for testing, but the vehicles seem to be more accurate when commented than previously.  So I agree with you - get rid of the function.